plugins/dependencies/src/test/resources/dependencies-mapping-validated.txt

```diff
@@ -747,6 +771,14 @@ io.pivotal.spring.cloud..spring-cloud-services-starter-config-client=Spring.spri
 io.pivotal.spring.cloud..spring-cloud-services-starter-service-registry=Spring.springCloud.serviceRegistry
 io.projectreactor..reactor-test=Spring.reactor.test
 io.projectreactor.kotlin..reactor-kotlin-extensions=Spring.reactor.kotlin
+io.reactivex..rxjava=ReactiveX.rxJava1
+io.reactivex..rxkotlin=ReactiveX.rxKotlin1
```

Comment:
This looks inconsistent with what you have defined in the code (where there's no "1" suffix). Please, revert the changes on the files that are edited by the tests if you are editing the dependency notations in the Kotlin code.

Since we'll likely not include these dependency notations, it'll be obsolete of course, but at least, you know the process a bit better now.